src/CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ include_directories(../include)
 
 find_package(CUDA REQUIRED)
 
+if(CUDA_VERSION VERSION_GREATER_EQUAL "9.0")
```

Comment:
`VERSION_GREATER_EQUAL` statement is enabled to use CMake 3.7 or newer.
Replace with VERSION_GREATER and VERSION_EQUAL for older versions.
https://cmake.org/cmake/help/v3.7/release/3.7.html#commands

Reply:
Should be fixed now

Reply:
Thank you!